examples/basic/lines/arcs.py

```diff
@@ -1,3 +1,25 @@
+''' A ac graph using pre-defined data points. This example
+demonstrates the use of the ``arc`` method to make a graph
+<<<<<<< HEAD
```

Comment:
Hi @KesharwaniArpita it looks like some diff noise accidentally got commited

Reply:
how can i resolve that?

Reply:
HI @KesharwaniArpita these are lines of text in the `.py` module. If you open up the file in a text editor or IDE you should see them and be able to edit them out. Basically, just make the file look like however it is supposed to look, and then push changes again

Reply:
![image](https://user-images.githubusercontent.com/107834813/229145987-f508f76e-55c7-40c5-a98e-9e33ad4db460.png)
@bryevdv ,Well, this is how it looks!

Reply:
@KesharwaniArpita what is the output of `git status`? The most likely explanation is that for some reason, the current  local version above has either not been committed, or not been pushed. 

If it is simpler, it would also be ok to close this PR and open a new "clean" PR.